tests/monitoring/metrics.go

```diff
@@ -124,6 +124,40 @@ var _ = Describe("[sig-monitoring]Metrics", decorators.SigMonitoring, func() {
 			metrics := fetchPrometheusMetrics(virtClient, query)
 			Expect(metrics.Data.Result).To(BeEmpty(), "Expected no workqueue_depth metrics for virt workloads")
 		})
+
+		It("kubevirt workqueue metrics should include controllers names", func() {
+			names := []string{
```

Comment:
TODO: we should add `virt-synchronization-controller` when Decentralized LV will be enabled by default, or GA